doc/source/whatsnew/v0.19.0.txt

```diff
@@ -433,6 +433,13 @@ Deprecations
 - ``as_recarray`` has been deprecated in ``pd.read_csv()`` and will be removed in a future version (:issue:`13373`)
 - top-level ``pd.ordered_merge()`` has been renamed to ``pd.merge_ordered()`` and the original name will be removed in a future version (:issue:`13358`)
 
+.. _whatsnew_0190.prior_deprecations:
+
+Removal of prior version deprecations/changes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- Removal of the legacy time rules (offset aliases), deprecated since 0.14.0 (this has been alias since 0.8.0) (GH13590).
+
```

Comment:
I would also add a blurb about what exactly these are (show an example), and point to the new docs as well.